policy-controller/core/src/outbound.rs

```diff
@@ -56,6 +57,7 @@ pub struct WeightedAddr {
     pub weight: u32,
     pub addr: IpAddr,
     pub port: NonZeroU16,
+    pub request_timeout: Option<time::Duration>,
```

Comment:
This struct is unused, because we don't yet support policy lookups for specific endpoints.  Even once we do, it's unclear how timeouts would get configured for an individual endpoint.  Probably okay to leave this off for now until we have a plan for how to support/configure it.